packages/task/test-resources/.theia/tasks.json

```diff
@@ -55,6 +55,57 @@
                     "dir"
                 ]
             }
+        },
+        {
+          "label": "Test compile problemMatcher",
+          "type": "shell",
+          "command": "tsc ${file}",
```

Comment:
I don't believe this task actually works as intended in the `test-resources` workspace, there are no typescript files to run `tsc` on.